### PR TITLE
Feature/comment

### PR DESCRIPTION
**Pull request by guswnsxodlf**
Add comment functions

### CODE REVIEW

File: kboard/board/views.py [hunk 1]
```diff
@@ -20,9 +20,14 @@ def post_list(request, board_id):
 
 
 def view_post(request, post_id):
-    post_ = Post.objects.get(id=post_id)
+    if request.method == 'POST':
```
Comment (darjeeling) [inline kboard/board/views.py]: why POST to view_post create comment? I think it's wired and too complex to understand

**Comment by guswnsxodlf** [comment]
I rebase commits :)

File: kboard/kboard/urls.py [hunk 2]
```diff
@@ -15,7 +15,6 @@
 """
 from django.conf.urls import url, include
 
-app_name = 'board'
```
Comment (darjeeling) [inline kboard/kboard/urls.py:18]: why is this line deleted?
